pages/TxKey.qml

```diff
@@ -222,7 +255,7 @@ Rectangle {
                 font.family: MoneroComponents.Style.fontRegular.name
                 font.pixelSize: 14
                 color: MoneroComponents.Style.defaultFontColor
-            }
+            }            
```

Comment:
looks unintended

Reply:
Oops. Not sure what happened there. Fixed on 2bf0dd840f907b406271863540c8efe002b2cdc3